plugins/live/src/scriptcommandlineparser.cpp

```diff
@@ -15,6 +15,7 @@
 ****************************************************************************/
 
 #include "scriptcommandlineparser_p.h"
+#include <stdexcept>
```

Comment:
We should rather replace `std::runtime_error` with `lv::Exception` and remote this header.